tdrs-backend/tdpservice/parsers/test/test_parse.py

```diff
@@ -1880,4 +1881,4 @@ def test_parse_cat_4_edge_case_file(cat4_edge_case_file, dfs):
 
     err = parser_errors.first()
     assert err.error_message == ("Every T1 record should have at least one corresponding T2 or T3 record with the "
-                                 "same RPT_MONTH_YEAR and CASE_NUMBER.")
+                                 "same Item 4 (Reporting Year and Month) and Item 6 (Case Number).")
```

Comment:
@elipe17 Out of curiosity if we wanted to add a reference to a knowledge center article like we did in the update indicator error would that require additional dev lift to implement for this type of error or is that easily appended to what we have already?

Reply:
@elipe17 and could you remind me whether we have a ticket anywhere in the backlog that would deliver "friendly names" for record types? e.g. T1 (Family)

Reply:
I don't think either of these questions block this ticket, just trying to keep my head around outstanding parsing work!

Reply:
@reitermb 

Re Q1: I don't see why it would be an issue to add a reference to the knowledge center. It might be a bit more challenging if we want a very specific reference that needs to be dynamically generated, but that also shouldn't be too hard to handle.

Re Q2: I am not aware of a ticket that provides friendly names for the record types.